src/main/java/net/dv8tion/jda/core/entities/Webhook.java

```diff
@@ -57,11 +58,12 @@
     TextChannel getChannel();
 
     /**
-     * The owner of this Webhook.
+     * The owner of this Webhook. This will be null for partial Webhooks, such as those retrieved from Audit Logs.
```

Comment:
We should mark `Webhook` as `IFakable` for this scenario. Fake webhooks would not be able to use `newClient()` or `getManager()`.